todo_app/app.py

```diff
@@ -1,25 +1,60 @@
-
+import json
 from flask import Flask, render_template, request, redirect, url_for, session
+from flask_login.utils import login_user, login_required
 from todo_app.data import session_items
 from todo_app.data.trello import Trello
 import requests
 import os
+import logging
+from flask_login import LoginManager, UserMixin, login_required, login_user, current_user
+from todo_app.data.user import User
+from oauthlib.oauth2 import WebApplicationClient
+
 
 def create_app():
     app = Flask(__name__)
     app.config.from_object('todo_app.flask_config.Config')
-        
+    login_manager = LoginManager()
+    login_manager.init_app(app)  
     trello=Trello()
-
+    #app.logger.setLevel(app.config['LOG_LEVEL'])
+    os.environ['OAUTHLIB_INSECURE_TRANSPORT']= "1"
```

Comment:
Rather than setting it here, we should just set it in your .env file where necessary; we _don't_ want to allow insecure communication in the production app for example

Reply:
done